package.json

```diff
@@ -98,7 +98,7 @@
     "a11y:generate": "./scripts/generateAccessibilitySpecs.js",
     "a11y:validate": "./scripts/validateAccessibilitySpecs.js",
     "svgIcons:validate": "./scripts/validateIconSvgSpecs.js",
-    "build": "cd packages/gestalt && yarn build && cd ../gestalt-datepicker && yarn build && cd ../eslint-plugin-gestalt && yarn build && cd ../gestalt-design-tokens && yarn build && cd ../../docs && yarn build",
+    "build": "cd packages/gestalt-design-tokens && yarn build && cd ../gestalt && yarn build && cd ../gestalt-datepicker && yarn build && cd ../eslint-plugin-gestalt && yarn build && cd ../../docs && yarn build",
```

Comment:
should we note it here as well? the build is probably the most important part where order matters, no?

Reply:
@AlbertCarreras Does a JSON file allow comments?